### PR TITLE
[ENG-1404] chore: use `displayName` instead of provider name

### PR DESCRIPTION
**Pull request by jatinsandilya**

### CODE REVIEW

File: src/components/ApiKeyAuth/ApiKeyAuthFlow/LandingContent.tsx [hunk 1]
```diff
@@ -41,7 +41,7 @@ export function LandingContent({
     <ApiKeyAuthCardLayout>
       <FormControl>
         <FormLabel my="6">
-          <Heading as="h4" size="md">{`Set up ${capitalize(provider)} integration`}</Heading>
+          <Heading as="h4" size="md">{`Set up ${providerName} integration`}</Heading>
```
Comment (dionlow) [inline src/components/ApiKeyAuth/ApiKeyAuthFlow/LandingContent.tsx:44]: is providerName guaranteed or should we fall back to the capitalize provider?

Comment (jatinsandilya) [reply src/components/ApiKeyAuth/ApiKeyAuthFlow/LandingContent.tsx:44]: `providerName` includes the logic to fallback to `capitalize(provider)` see: https://github.com/amp-labs/react/pull/473/files#diff-adac1f48e166b537f09a13bb4faa2fb2f865a822cd93a9891504e1961a2e0e3fL37

**Comment by jatinsandilya** [comment]
@dionlow i've tested this with the demo app repo but I'm wondering if there are other/better ways to be sure everything works alright since this touches a lot of the react sdk code ?

File: src/components/ApiKeyAuth/ApiKeyAuthFlow/LandingContent.tsx (hunk 1 above)
Comment (dionlow) [reply src/components/ApiKeyAuth/ApiKeyAuthFlow/LandingContent.tsx:44]: I see thanks for clarifying.

**Review by dionlow — APPROVED** [review]
lgtm

**Comment by dionlow** [comment]
@jatinsandilya We currently use the demo app mostly to test and haven't had too many issues yet. Are you asking in terms of testing the build or the whole package?

**Pull request merged**